test/browser_tests/spec_suites/header.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var Header = require( '../shared_objects/header.js' );
+
+describe( 'The Header Component', function() {
+  var _sharedObject;
+
+  beforeAll( function() {
+    _sharedObject = new Header();
+    _sharedObject.get();
+
+    function getStylesheets() {
+      var sheets = document.styleSheets;
+      var hrefs = [];
+
+      for( i = 0; i < sheets.length; i++ ) {
+        hrefs.push( sheets[i].href );
+      }
+
+      hrefs.push( 'http://google.com/foo.css' );
```

Comment:
Yes, there's no reason to push an extra stylesheet url to the array, was just testing to make sure it looped through them all later.